python/jiminy_py/src/jiminy_py/simulator.py

```diff
@@ -255,7 +283,7 @@ def robot_state(self) -> jiminy.RobotState:
 
     @property
     def is_viewer_available(self) -> bool:
-        """Returns whether a viewer instance associated with the robot
+        """Returns whether a viewer instance associated with the first robot
```

Comment:
`associated with the ongoing simulation.` should be fine, and closer to the question the user is trying to answer.